client/my-sites/site-settings/traffic/main.jsx

```diff
@@ -33,6 +37,15 @@ const SiteSettingsTraffic = ( {
 		<SiteSettingsNavigation site={ site } section="traffic" />
 
 		<SeoSettingsHelpCard />
+		{ jetpackSettingsUiSupported &&
```

Comment:
Nah, I'd rather not do it this way. Soon we'll also have the AMP card in Traffic - this is addressed in #12407. It will also require that check, so it would make sense to keep it here. Also, why mount an entire new component if we can avoid that? 😉